Composer/packages/client/src/pages/home/CardWidget.tsx

```diff
@@ -74,10 +76,29 @@ export const CardWidget: React.FC<CardWidgetProps> = ({
     const rect = e.currentTarget.getBoundingClientRect();
     const aspectRatio = rect.width / rect.height;
```

Comment:
@zhixzhan we can use the screen dimensions (media query) to set the width of the element for different breakpoints. I dont think we need to adjust the height.